Makefile

```diff
@@ -6,7 +6,7 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --tty --interactive --rm $(IMAGE_NAME) bundle exec rake
+	docker run --tty --interactive --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
```

Comment:
@pbrisbin Actually, I removed the docker file changes. They fall into category of driveby and also independent of any other change here result in an exception getting raised that looks like:

```
LoadError: load error: bundler/setup -- java.lang.RuntimeException: Unhandled IOException: java.io.IOException: unhandled errno: Read-only file system
  require at org/jruby/RubyKernel.java:939
  require at /opt/jruby/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:128
    <top> at /usr/src/app/lib/cc/engine/duplication.rb:1
  require at org/jruby/RubyKernel.java:939
   (root) at /opt/jruby/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1
    <top> at /opt/jruby/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:54
```